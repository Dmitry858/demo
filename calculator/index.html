<!DOCTYPE html>
<html lang="ru">
<head>
    <meta charset="UTF-8">
    <title>Калькулятор услуг</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="./css/style.css">
</head>
<body>
    <div class="container">
        <h1 class="title">Калькулятор</h1>

        <div class="steps-points">
            <ul>
                <li class="point current">Услуга</li>
                <li class="point">Адрес</li>
                <li class="point">Площадь</li>
                <li class="point">Ваши контакты</li>
                <li class="point">Расчёт</li>
            </ul>
        </div>

        <div class="steps-points-mobile">
            <div class="point-mobile-current">Услуга</div>
            <ul class="points-mobile-list hidden">
                <li class="point-mobile current">Услуга</li>
                <li class="point-mobile">Адрес</li>
                <li class="point-mobile">Площадь</li>
                <li class="point-mobile">Ваши контакты</li>
                <li class="point-mobile">Расчёт</li>
            </ul>
        </div>

        <form class="steps-wrap" action="">
            <div class="step step-one active">
                <h2 class="step-title">Укажите какие услуги вам нужны</h2>

                <div class="element-wrap">
                    <img src="./img/service1.jpg" alt="">
                    <input type="checkbox" id="service1" name="service1" value="Подготовка технического плана">
                    <label for="service1">Подготовка технического плана</label>
                </div>

                <div class="element-wrap">
                    <img src="./img/service2.jpg" alt="">
                    <input type="checkbox" id="service2" name="service2" value="Вынос границ участка в натуру">
                    <label for="service2">Вынос границ участка в натуру</label>
                </div>

                <div class="element-wrap">
                    <img src="./img/service3.jpg" alt="">
                    <input type="checkbox" id="service3" name="service3" value="Подготовка межевого плана">
                    <label for="service3">Подготовка межевого плана</label>                    
                </div>

                <div class="element-wrap">
                    <img src="./img/service4.jpg" alt="">
                    <input type="checkbox" id="service4" name="service4" value="Исполнительная съёмка">
                    <label for="service4">Исполнительная съёмка</label>                    
                </div>

                <div class="element-wrap">
                    <img src="./img/service5.jpg" alt="">
                    <input type="checkbox" id="service5" name="service5" value="Топографическая съёмка">
                    <label for="service5">Топографическая съёмка</label>                   
                </div>

                <div class="element-wrap">
                    <a class="question" href="#">?</a>
                    <a class="btn btn-forward" href="#">Дальше</a>
                </div>

            </div>

            <div class="step step-two hidden">
                <h2 class="step-title">Укажите адрес работ</h2>

                <div class="element-wrap">
                    <input type="text" placeholder="Например, г. Москва, ул. Примерная, 1" required="required">
                    <a class="question" href="#">?</a>
                </div>

                <div class="nav-links">
                    <a class="btn btn-back" href="#">Назад</a>
                    <a class="btn btn-forward" href="#">Дальше</a>
                </div>
            </div>

            <div class="step step-three hidden">
                <h2 class="step-title">Укажите площадь объекта в сотках</h2>

                <div class="element-wrap">
                    <input type="number" value="" required="required">
                    <input type="checkbox" class="custom-checkbox" id="docs" name="docs" value="Подать документы" checked required="required">
                    <label for="docs">Подать документы</label>  
                </div>

                <div class="element-wrap">
                    <a class="question" href="#">?</a>
                </div>

                <div class="nav-links">
                    <a class="btn btn-back" href="#">Назад</a>
                    <a class="btn btn-forward" href="#">Дальше</a>
                </div>
            </div>

            <div class="step step-four hidden">
                <h2 class="step-title">Оставьте свои данные для расчёта суммы</h2>

                <div class="element-wrap">
                    <div class="inputs-group">
                        <input type="text" placeholder="Имя" required="required">
                        <input type="text" placeholder="Телефон" required="required">
                        <textarea name="comment" placeholder="Комментарий"></textarea>
                    </div>

                    <div class="checkboxes-group">
                        <input type="checkbox" class="custom-checkbox" id="call" name="call" value="Позвонить на телефон" checked>
                        <label for="call">Позвонить на телефон</label> 

                        <input type="checkbox" class="custom-checkbox" id="whatsapp" name="whatsapp" value="Отправить сообщение в WhatsApp">
                        <label for="whatsapp">Отправить сообщение в WhatsApp</label> 

                        <input type="checkbox" class="custom-checkbox" id="viber" name="viber" value="Отправить сообщение в Viber">
                        <label for="viber">Отправить сообщение в Viber</label> 
                    </div> 
                </div>

                <div class="element-wrap">
                    <a class="question" href="#">?</a>
                </div>

                <div class="element-wrap w100">
                    <input type="checkbox" class="custom-checkbox" id="agree" name="agree" value="Согласие с политикой персональных данных" checked required="required">
                    <label for="agree">Согласие с <a class="agreement" href="#">Политикой персональных данных</a></label>  
                </div>

                <div class="nav-links">
                    <a class="btn btn-back" href="#">Назад</a>
                    <button type="submit" class="btn btn-forward" href="#">Дальше</button>
                </div>
            </div>

            <div class="step step-five hidden">
                <h2 class="step-title">Примерная сумма проекта</h2>

                <div class="result">
                    <div class="user-data">
                        <p><span>1. Услуга:</span> <span>Исполнительная съёмка</span></p>
                        <p><span>По адресу:</span> <span>Москва, МО</span></p>
                        <p><span>Площадь:</span> <span>21 сотка</span></p>
                    </div>
                    <div class="price">
                        <p><span>Ориентировочная стоимость услуги:</span> <span class="crossed-out">10 000 р.</span></p>
                        <p><span>Скидка:</span> <span class="primary-color">10%</span></p>
                        <p><span>Стоимость услуги с учетом скидки:</span> <span class="primary-color">9 000 р.</span></p>
                    </div>
                    <div class="discount">
                        <p><span>Акция:</span> <span class="primary-color larger-font">Геоакция на Март</span></p>
                        <div class="total-price-link">
                            <a href="#">Ориентировочный бюджет проекта с учетом скидки:</a>      
                            <a class="down-arrow" href="#">&darr;</a>
                        </div>
                    </div>
                </div>

                <div class="result">
                    <div class="user-data">
                        <p><span>2. Услуга:</span> <span>Исполнительная съёмка</span></p>
                        <p><span>По адресу:</span> <span>Москва, МО</span></p>
                        <p><span>Площадь:</span> <span>21 сотка</span></p>
                    </div>
                    <div class="price">
                        <p><span>Ориентировочная стоимость услуги:</span> <span class="primary-color">10 000 р.</span></p>
                    </div>
                    <div class="discount discount-empty">
                    </div>
                </div>

                <div class="result">
                    <div class="user-data">
                        <p><span>3. Услуга:</span> <span>Исполнительная съёмка</span></p>
                        <p><span>По адресу:</span> <span>Москва, МО</span></p>
                        <p><span>Площадь:</span> <span>21 сотка</span></p>
                    </div>
                    <div class="price">
                        <p><span>Ориентировочная стоимость услуги:</span> <span class="crossed-out">10 000 р.</span></p>
                        <p><span>Скидка:</span> <span class="primary-color">10%</span></p>
                        <p><span>Стоимость услуги с учетом скидки:</span> <span class="primary-color">9 000 р.</span></p>
                    </div>
                    <div class="discount">
                        <p><span>Акция:</span> <span class="primary-color larger-font">Скидка на всё 10%</span></p>
                    </div>
                </div>

                <div class="total-price">
                    <p><span>Ориентировочный бюджет проекта с учётом скидки:</span> <span>15 990 р.</span></p>                    
                </div>

                <div class="reset-form">
                    <button type="reset" class="btn btn-reset" href="#">Сделать ещё один расчёт</button>
                </div>
            </div>

        </form>
          
    </div> 

    <script src="./js/common.js"></script>
</body>
</html>