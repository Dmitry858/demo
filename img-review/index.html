<!DOCTYPE html>
<html lang="ru">
  <head>
    <title>UI</title>
    <meta charset="utf-8">
    <link rel="stylesheet" href="./css/reset.css">
    <link rel="stylesheet" href="./css/styles.css">
  </head>

  <body>
    <div class="wrap app" data-state="initial">
      <ul class="menu">
        <li class="menu__item drag"></li>

        <li class="menu__item burger invisible">
          <i class="burger-icon"></i>
        </li>

        <li class="menu__item mode new">
          <i class="menu__icon new-icon"></i>
          <span class="menu__item-title">Загрузить<br>новое</span>
        </li>

        <li class="menu__item mode comments invisible" data-state="">
          <i class="menu__icon comments-icon"></i>
          <span class="menu__item-title">Комментарии</span>
        </li>
        <li class="menu__item tool comments-tools invisible">
          <span class="menu__toggle-bg">
            <input class="menu__toggle" type="radio" name="toggle" value="on" id="comments-on" checked><label class="menu__toggle-title menu__toggle-title_on" for="comments-on">Показать<br>комментарии</label>
            <input class="menu__toggle" type="radio" name="toggle" value="off" id="comments-off"><label class="menu__toggle-title menu__toggle-title_off" for="comments-off">Скрыть<br>комментарии</label>
            <span class="menu__toggle-bttn"></span>
          </span>
        </li>

        <li class="menu__item mode draw invisible" data-state="">
          <i class="menu__icon draw-icon"></i>
          <span class="menu__item-title">Рисовать</span>
        </li>
        <li class="menu__item tool draw-tools invisible">
          <input class="menu__color red" type="radio" name="color" value="#ea5d56"><span></span>
          <input class="menu__color yellow" type="radio" name="color" value="#f3d135"><span></span>
          <input class="menu__color green" type="radio" name="color" value="#6cbe47" checked><span></span>
          <input class="menu__color blue" type="radio" name="color" value="#53a7f5"><span></span>
          <input class="menu__color purple" type="radio" name="color" value="#b36ade"><span></span>
        </li>

        <li class="menu__item mode share invisible" data-state="">
          <i class="menu__icon share-icon"></i>
          <span class="menu__item-title">Поделиться</span>
        </li>
        <li class="menu__item tool share-tools invisible">
            <input class="menu__url" type="text" value="">
            <input class="menu_copy" type="button" value="Копировать">
        </li>
        <input class="hidden" type="file">
      </ul>

      <img class="current-image" src="">
      
      <div class="error invisible">
        <h4 class="error__header">Ошибка</h4>
        <p class="error__message">Неверный формат файла. Пожалуйста, выберите изображение в формате .jpg или .png.</p>
      </div>
      
      <div class="image-loader invisible">
        <div class="loader">
          <span></span>
          <span></span>
          <span></span>
          <span></span>
          <span></span>
        </div>
        <p class="image-loader__message">Загружаю изображение...</p>
      </div>      

    </div>
    
    <script src="./js/solution.js"></script>
  </body>

</html>