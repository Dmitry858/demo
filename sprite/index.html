<!doctype html>
<html lang="ru">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Генератор спрайтов</title>
    <link rel="stylesheet" href="css/style.css">
</head>
<body>
    <div class="sprite-generator" id="generator">
        <div class="sprite-generator__title">
            <h2>Генератор спрайтов</h2>   
        </div>
        <p>Добавьте изображения и нажмите на кнопку «Сгенерировать спрайт»</p>
        <p>Размер каждого изображения не должен превышать 30 Кб. Максимальное количество изображений - 81</p>
        <input type="file" multiple class="sprite-generator__upload">
        <span class="error hidden"></span>
        <div class="images__added-count">Изображений в спрайте:
            <span class="images__added-count-value">0</span>
        </div>
        <!-- <canvas id="canvas"></canvas> -->
        <button class="sprite-generator__generate">Сгенерировать спрайт</button>
        <button class="sprite-generator__clear">Очистить генератор</button>
        <div class="sprite-generator__result">

            <div class="sprite-generator__preview">
                <img alt="" class="sprite-generator__result-image">
            </div>
            <textarea class="sprite-generator__code" cols="30" rows="15" placeholder="Здесь будет ваш CSS" readonly></textarea>
        </div>
    </div>
    <script src="js/main.js"></script>
</body>
</html>
