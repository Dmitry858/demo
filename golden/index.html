<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Golden Studio</title>
	<link href="css/styles.css" rel="stylesheet">
	<link href="https://fonts.googleapis.com/css?family=Cookie" rel="stylesheet">
	<link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,600,700&amp;subset=cyrillic" rel="stylesheet">
	<link href="https://fonts.googleapis.com/css?family=Vollkorn:400,400i" rel="stylesheet">
	<link href="https://fonts.googleapis.com/css?family=Roboto+Slab&amp;subset=cyrillic-ext" rel="stylesheet">
</head>
<body>
	<header>
		<div class="wrap">
			<p class="logo">Golden</p>
			<nav>
				<ul>
					<li>
						<a href="">Home</a>
					</li>
					<li>
						<a href="">Services</a>
					</li>
					<li>
						<a href="">Portfolio</a>
					</li>
					<li>
						<a href="">About</a>
					</li>
					<li>
						<a href="">Contact</a>
					</li>
				</ul>
			</nav>
			<h1>Welcome to our studio!</h1>
			<p class="title">It's nice to meet you</p>
			<a class="link" href="">Tell me more</a>
		</div>
	</header>

	<section id="services">
		<div class="wrap">
			<h2>Services</h2>
			<p class="subtitle">Proin iaculis purus consequat sem cure.</p>
			<div class="service">
				<div class="service-commerce"></div>
				<h3>E-commerce</h3>
				<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
			</div>
			<div class="service center">
				<div class="service-web"></div>
				<h3>Responsive web</h3>
				<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
			</div>
			<div class="service">
				<div class="service-security"></div>
				<h3>Web security</h3>
				<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
			</div>
		</div>
	</section>

	<section id="portfolio">
		<div class="wrap">
			<h2>Our portfolio</h2>
			<p class="subtitle">Proin iaculis purus consequat sem cure.</p>
			<div class="item">
				<a href="">
					<img src="img/image1.jpg" alt="">
					<h3>Ebony &amp; Ivory</h3>
					<p class="item-subtitle">Branding</p>
				</a>
			</div>
			<div class="item">
				<a href="">
					<img src="img/image2.jpg" alt="">
					<h3>Smart stationary</h3>
					<p class="item-subtitle">Print design</p>
				</a>
			</div>
			<div class="item">
				<a href="">
					<img src="img/image3.jpg" alt="">
					<h3>Clever poster</h3>
					<p class="item-subtitle">Print design</p>
				</a>
			</div>
			<div class="item">
				<a href="">
					<img src="img/image4.jpg" alt="">
					<h3>Vinyl record</h3>
					<p class="item-subtitle">Product mock-up</p>
        </a>
			</div>
			<div class="item">
				<a href="">
					<img src="img/image5.jpg" alt="">
					<h3>Treehouse template</h3>
					<p class="item-subtitle">Web design</p>
        </a>
			</div>
			<div class="item">
				<a href="">
					<img src="img/image6.jpg" alt="">
					<h3>Burned logo</h3>
					<p class="item-subtitle">Branding</p>
        </a>
			</div>
	  </div>
	</section>

	<section id="about">
		<div class="wrap">
			<h2>About us</h2>
			<p class="subtitle">Proin iaculis purus consequat sem cure.</p>
			<div class="wrap-items">
				<div class="about-item">
					<div class="about-left">
						<h4>July 2010</h4>
						<p class="about-item-subtitle">Our humble beginnings</p>
						<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
					</div>
					<div class="about-center">
						<img src="img/about-img1.png" alt="">
					</div>
					<div class="about-right"></div>
				</div>
				<div class="about-item">
					<div class="about-left"></div>
					<div class="about-center">
						<img src="img/about-img2.png" alt="">
					</div>
					<div class="about-right">
						<h4>January 2011</h4>
						<p class="about-item-subtitle">Facing startup battles</p>
						<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
					</div>
				</div>
				<div class="about-item">
					<div class="about-left">
						<h4>December 2012</h4>
						<p class="about-item-subtitle">Enter the dark days</p>
						<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni.</p>
					</div>
					<div class="about-center">
						<img src="img/about-img3.png" alt="">
					</div>
					<div class="about-right"></div>
				</div>
				<div class="about-item">
					<div class="about-left"></div>
					<div class="about-center">
						<img src="img/about-img4.png" alt="">
					</div>
					<div class="about-right">
						<h4>February 2014</h4>
						<p class="about-item-subtitle">Our triumph</p>
						<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean.</p>
					</div>
				</div>
				<div class="about-item">
					<div class="about-center">
						<h4>Our story continues ...</h4>
					</div>
				</div>
			</div>
		</div>
	</section>

	<section id="team">
		<div class="wrap">
			<h2>Our amazing team</h2>
			<p class="subtitle">Proin iaculis purus consequat sem cure.</p>
			<div class="person">
				<div class="person-photo"></div>
				<p>Kimberly Thompson</p>
				<p class="prof">Marketer</p>
				<a href="">
					<div class="icon-twitter"></div>
				</a>
        <a href="">
					<div class="icon-facebook"></div>
				</a>
        <a href="">
					<div class="icon-pinterest"></div>
				</a>
			</div>
			<div class="person">
				<div class="person-photo"></div>
				<p>Rico Massimo</p>
				<p class="prof">Coder</p>
				<a href="">
					<div class="icon-twitter"></div>
				</a>
				<a href="">
					<div class="icon-facebook"></div>
				</a>
        <a href="">
					<div class="icon-google"></div>
				</a>
			</div>
			<div class="person">
				<div class="person-photo"></div>
				<p>Uku Mason</p>
				<p class="prof">Graphic designer</p>
				<a href="">
					<div class="icon-facebook"></div>
				</a>
				<a href="">
					<div class="icon-pinterest"></div>
				</a>
			</div>
			<p class="team-text">Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
		</div>
	</section>

	<div class="logos">
		<img src="img/logo-partner1.jpg" alt="">
		<img src="img/logo-partner2.png" alt="">
		<img src="img/logo-partner3.jpg" alt="">
		<img src="img/logo-partner4.jpg" alt="">
	</div>

	<section id="contact">
		<div class="wrap">
			<h2>Contact us</h2>
			<p class="subtitle">Proin iaculis purus consequat sem cure.</p>
			<form action="send.php">
				<div class="fields">
					<div class="field">
						<input type="text" name="name" required placeholder="Your name *">
					</div>
					<div class="field">
						<input type="text" name="email" required placeholder="Your e-mail *">
					</div>
					<div class="field">
						<input type="text" name="subject" placeholder="Subject">
					</div>
				</div>
				<div class="fields">
					<textarea name="message" required placeholder="Your message *"></textarea>
				</div>
        <input type="submit" value="Send message">
			</form>
		</div>
	</section>

	<footer>
		<div class="wrap">
			<div class="copyright">
				<p>&copy; Copyright 2018 Golden</p>
			</div>
			<div class="social">
				<a href="">
					<div class="icon-twitter"></div>
				</a>
				<a href="">
					<div class="icon-facebook"></div>
				</a>
    		<a href="">
					<div class="icon-pinterest"></div>
				</a>
				<a href="">
					<div class="icon-google"></div>
				</a>
			</div>
		</div>
	</footer>
</body>
</html>